docs/EN_US/HPCCClientTools/CT_Mods/CT_ECL_CLI.xml

```diff
@@ -150,7 +150,8 @@
               <row>
                 <entry>zapgen</entry>
 
-                <entry>returns the zap file name from the given WUID.</entry>
+                <entry>generate and download the entire zap file for the given
+                WUID.</entry>
```

Comment:
Sorry, my bad, I think I used the word "entire" but now that I see it I think that word should be removed.